engine/src/physics/plugin.cpp

```diff
@@ -43,6 +46,27 @@ CUBOS_REFLECT_IMPL(Impulse)
         .build();
 }
 
+CUBOS_REFLECT_EXTERNAL_DECL(CUBOS_EMPTY, PhysicsMaterial::MixProperty);
```

Comment:
Reflection declarations should stay in header files - otherwise, users of this enum won't be able to reflect it if necessary. You can move it below the cubos::engine namespace in the header. Only the definition should be here.